architects/demo/knowledge_article.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="knowledge_article_3" model="knowledge.article">
+        <field name="is_published" eval="True"/>
+    </record>
```

Comment:
I thought we reached the agreement that knowledge articles shouldn't be published at all.
```suggestion
```